lib/ui/src/modules/ui/components/theme.js

```diff
@@ -1,7 +1,7 @@
 export const baseFonts = {
   fontFamily: `
-    -apple-system, ".SFNSText-Regular", "San Francisco", "Roboto",
-    "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
+    "Cantarell", "Fira Sans", "Droid Sans", "Arial", sans-serif
```

Comment:
I think `Helvetica Neue` is still necessary for OS X 10.11 and below when not in Chrome:
https://bitsofco.de/the-new-system-font-stack/